src/App.js

```diff
@@ -1,12 +1,47 @@
 /* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] OPTIONAL 
You have **disabled linter check** on your code. Please enable linter check on your code to ensure a cleaner syntax.

Reply:
Okay, I'll arrange that.